packages/components/tests/integration/components/hds/button-set/index-test.js

```diff
@@ -0,0 +1,17 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | hds/button-set/index', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders the component', async function (assert) {
+    await render(hbs`<Hds::ButtonSet />`);
+    assert.dom(this.element).exists();
+  });
+  test('it should render with a CSS class that matches the component name', async function (assert) {
+    await render(hbs`<Hds::ButtonSet id="test-button-set" />`);
+    assert.dom('#test-button-set').hasClass('hds-button-set');
+  });
+});
```

Comment:
Are there any additional tests which would be helpful to add?

Reply:
If it was me, probably I would do something like this:
```
  test('it should render a child button', async function (assert) {
    await render(
      hbs`<Hds::ButtonSet id="test-button-set"><Hds::Button @text="my button" /></Hds::ButtonSet>`
    );
    assert.dom('#test-button-set .hds-button').exists();
  });
```
It may be redundant, but essentially it tests that the content is yielded.